src/js/view/view.js

```diff
@@ -259,7 +259,8 @@ function View(_api, _model) {
             addClass(_playerElement, 'jw-ie');
         }
 
-        handleColorOverrides(_model.get('id'), _model.get('skinColors'));
+        const skinColors = normalizeSkin(_model.get('skin'));
```

Comment:
What happens when `_model.change('skin', onSkinChange, this);` is called and `_model.get('skin')` is undefined? Are we getting a `jw-skin-undefined` class added to the player?